resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,420 @@
+52316|Ayutla
+52317|Autlan De Navarro
+52319|Tuxpan
+52321|Grullo, El
+52322|Puerto Vallarta
+52323|Santiago Ixcuintla
+52324|Ixtlan Del Rio
+52325|Acaponeta
+52326|Atemajac De Brizuela
+52327|Compostela
+52328|Vista Hermosa De Negrete
+52329|Valle De Banderas
+5233|Guadalajara
+52341|Ciudad Guzman
+52342|Sayula
+52343|Juanacatlan
+52344|Yahualica De Gonzalez Gallo
+52345|Ayotlan
+52346|Teocaltiche
+52347|San Miguel El Alto
+52348|Arandas
+52349|Juchitlan
+52351|Zamora De Hidalgo
+52352|Piedad De Cabadas, La
+52353|Sahuayo De Morelos
+52354|Reyes De Salgado, Los
+52355|Tangancicuaro De Arista
+52356|Yurecuaro
+52357|Casimiro Castillo
+52358|Tamazula De Gordiano
+52359|Numaran
+52371|Tecalitlan
+52372|Teocuitatlan De Corona
+52373|Zapotlanejo
+52374|Tequila
+52375|Ameca
+52376|Chapala
+52377|Cocula
+52378|Tepatitlan De Morelos
+52381|San Jose De Gracia
+52382|Mazamitla
+52383|Santiago Tangamandapio
+52384|Tala
+52385|San Martin De Hidalgo
+52386|Magdalena
+52387|Acatlan De Juarez
+52388|Mascota
+52389|Tecuala
+52391|Atotonilco El Alto
+52392|Ocotlan
+52393|Barca, La
+52394|Cotija De La Paz
+52395|San Juan De Los Lagos
+52411|Cortazar
+52412|Comonfort
+52413|Apaseo El Alto
+52414|Tequisquiapan
+52415|San Miguel De Allende
+52417|Acambaro
+52418|Dolores Hidalgo Cuna De La Indep. Nal.
+52419|San Jose Iturbide
+52421|Jerecuaro
+52422|Ario De Rosales
+52423|Paracho De Verduzco
+52424|Coalcoman De Vazquez Pallares
+52425|Nueva Italia De Ruiz
+52426|Aguililla
+52427|San Juan Del Rio
+52428|San Felipe
+52429|Abasolo
+52431|Jalostotitlan
+52432|Ciudad Manuel Doblado
+52433|Sombrerete
+52434|Patzcuaro
+52435|Huetamo De Nuñez
+52436|Zacapu
+52437|Tlaltenango De Sanchez Roman
+52438|Puruandiro
+52441|Ezequiel Montes
+52442|Santiago De Queretaro
+52443|Morelia
+52444|San Luis Potosi
+52445|Moroleon
+52447|Maravatio De Ocampo
+52448|Pedro Escobedo
+52449|Aguascalientes
+52451|Zinapecuaro De Figueroa
+52452|Uruapan
+52453|Apatzingan De La Constitucion
+52454|Quiroga
+52455|Cuitzeo Del Porvenir
+52456|Valle De Santiago
+52457|Valparaiso
+52458|Ojocaliente
+52459|Tacambaro De Codallos
+52461|Celaya
+52462|Irapuato
+52463|Jalpa
+52464|Salamanca
+52465|Rincon De Romos
+52466|Salvatierra
+52467|Juchipila
+52468|San Luis De La Paz
+52469|Penjamo
+52471|Purepero De Echaiz
+52472|Silao
+52473|Guanajuato
+52474|Lagos De Moreno
+52475|Encarnacion De Diaz
+52476|San Francisco Del Rincon
+52477|Leon De Los Aldama
+52478|Victor Rosales
+52481|Ciudad Valles
+52482|Ciudad Del Maiz
+52483|Tamazunchale
+52485|Santa Maria Del Rio
+52486|Cerritos
+52487|Rioverde
+52488|Matehuala
+52489|Xilitla
+52492|Zacatecas
+52493|Fresnillo
+52494|Jerez De Garcia Salinas
+52495|Calvillo
+52496|Loreto
+52498|Rio Grande
+52499|Villanueva
+5255|Cuauhtemoc
+52588|Villa Del Carbon
+52591|Zumpango De Ocampo
+52592|Otumba De Gomez Farias
+52593|Huehuetoca
+52594|Acolman De Nezahualcoyotl
+52595|Texcoco De Mora
+52596|Reyes Acozac, Los
+52597|Amecameca De Juarez
+52599|Apaxco De Ocampo
+52612|Paz, La
```

Comment:
(same for all the names with a comma in them: I think it'll look much better to the user if they see a number being reported in "La Placename" rather than "Placename, La". Same for El etc.)